index.html

```diff
@@ -936,24 +936,36 @@ <h4>Text</h4>
                   <p>In <a>scripts</a> that have very little variation in source language,
                     the <a><code>daptm:langSrc</code></a> attribute can be set on the root element
                     and omitted from <code>&lt;p&gt;</code> elements except where its value
-                    differs.
-                    Care should be taken if using this approach, especially when moving between
+                    differs.</p>
```

Comment:
It may be good to add that if there is no `daptm:langSrc` is set on `<tt>` and `<p>` (e.g. it is not used at all in the document), all text objects signal that they have no inherent language.

Reply:
Done in aad01da